docs/source/managed-federation/setup.mdx

```diff
@@ -3,30 +3,65 @@ title: Setting up managed federation
+- Creating an Apollo account
+- Creating a graph in Studio
+- Installing and authenticating the Rover CLI
 
-> In the [Register your schema](https://www.apollographql.com/docs/studio/getting-started/#3-register-your-schema) step, make sure you follow the instructions for a GraphQL server that uses Apollo Federation.
+Then return here.
 
-## 2. Register all subgraph schemas
+## 2. Publish all subgraph schemas
 
-<RegisterFederatedCli />
+In a supergraph architecture, each of your [subgraphs](../building-supergraphs/subgraphs-overview/) uses the Rover CLI to publish its schema to GraphOS:
 
-## 3. Modify the gateway (if necessary)
+```mermaid
+graph LR;
+  subgraph " "
+  router([Router]);
+  subgraphA[Products subgraph];
+  subgraphB[Reviews subgraph];
+  subgraphC[Inventory subgraph];
+  end
+  registry{{GraphOS<br/>Schema Registry}};
+  router --- subgraphA & subgraphB & subgraphC;
+  subgraphA & subgraphB & subgraphC -.->|<code>rover subgraph publish</code>| registry;
+  class registry secondary;
+```
+
+**Do the following for each of your subgraphs**:
+
+1. Obtain the following values, which are required for the `rover subgraph publish` command:
+
+    * The name that uniquely identifies the subgraph within your graph (e.g., `products`).
+    * The URL that your router will use to communicate with the subgraph (e.g., `http://products-graphql.svc.cluster.local:4001/`).
+
+2. Run the `rover subgraph publish` command, providing it your subgraph's schema in one of the ways shown:
+
+    ```bash
+    # Provide a local .graphql file path
+    rover subgraph publish my-graph@my-variant --name products --routing-url http://products-graphql.svc.cluster.local:4001/ --schema ./schema.graphql 
 
-> This section assumes you are using Apollo Server with the `@apollo/gateway` library as your gateway.
+    # Provide an introspection result via stdin
+    rover subgraph introspect http://localhost:4000 | rover subgraph publish my-graph@my-variant --name products --routing-url http://products-graphql.svc.cluster.local:4001/ --schema -
+    ```
 
-If you've already set up Apollo Federation _without_ Apollo Studio, the constructor of your `ApolloGateway` instance probably includes a `supergraphSdl` option, like this:
+As you register your subgraph schemas, the schema registry attempts to **compose** their latest versions into a single **supergraph schema**. Whenever composition succeeds, your managed router can fetch the latest supergraph schema from GraphOS.
+
+You can also manually fetch your latest supergraph schema with the `rover supergraph fetch` command, or retrieve it from your graph's **Schema > SDL** tab in GraphOS Studio.
+
+## 3. Modify your router's startup command
+
+<ExpansionPanel title="Using Apollo Gateway?">
+
+If you've already set up Apollo Federation _without_ conneting your gateway to GraphOS, the constructor of your `ApolloGateway` instance probably includes a `supergraphSdl` option, like this:
```

Comment:
```suggestion
If you've already set up Apollo Federation _without_ connecting your gateway to GraphOS, the constructor of your `ApolloGateway` instance probably includes a `supergraphSdl` option, like this:
```

Reply:
I'm disappointed our spell checker didn't catch this, I had to double check that it was working at all! It is working, and it thinks that `conneting` is a word.

Reply:
Gasp! I am ashamed 😞